gips/data/daymet/daymet.py

```diff
@@ -133,24 +139,35 @@ def __init__(self, filename):
 
 
     @classmethod
+    def query_provider(cls, asset, tile, date):
+        """Determine availability of data for the given (asset, tile, date).
+
+        Returns (basename, url) on success; (None, None) otherewise.
+        Something of a no-op for daymet as the data is daily for the
+        entire run of the dataset and the URLs are deterministic.
+        """
+        source = cls._assets[asset]['source']
+        url = (cls._assets[asset]['url'] + '/%s') % (date.year, tile, date.year, source)
+        bn = cls._basename_pat.format(asset, tile, date.year, date.strftime('%j'))
+        return (bn, url)
+
+
+    @classmethod
     def fetch(cls, asset, tile, date):
-        """ Get this asset for this tile and date (using OpenDap service) """
-        url = cls._assets[asset].get('url', '') % (date.year, tile, date.year)
-        source = cls._assets[asset]['source'] 
-        loc = "%s/%s" % (url, source)
-        print loc
-        dataset = open_url(loc)
+        """Fetch a daymet asset and convert it to a gips-friendly format."""
+        asset_bn, url = cls.query_provider(asset, tile, date)
```

Comment:
It should once the date-checking is added to `query_provider`; right now it can't return Nones I don't think.